Source/WebCore/Headers.cmake

```diff
@@ -2047,28 +2047,29 @@ set(WebCore_PRIVATE_FRAMEWORK_HEADERS
     platform/graphics/angle/ANGLEUtilities.h
     platform/graphics/angle/GraphicsContextGLANGLE.h
 
-    platform/graphics/controls/ButtonPart.h
-    platform/graphics/controls/ColorWellPart.h
+    platform/graphics/controls/ButtonAppearance.h
```

Comment:
I have an alternative proposal at https://github.com/WebKit/WebKit/pull/23713 which doesn't require refactoring the code at all.